src/lib/server/runtime-config.ts

```diff
@@ -98,6 +98,7 @@ const baseConfig = {
 	collectIPOnDeliverylessOrders: false,
 	isBillingAddressMandatory: false,
 	disableLanguageSelector: false,
+	disableCartIcon: false,
```

Comment:
`hideCartInToolbar` is more appropriate (if that's all we want to do)

cc @Tirodem to confirm that's all we want to do